src/BlockchainServiceAPIClient.js

```diff
@@ -0,0 +1,132 @@
+const axios = require('axios');
+const schedule = require('node-schedule');
+
+const blockchainApiURL = process.env.BLOCKCHAIN_SERVICE_URL;
+const authorizationToken = Buffer.from(`${process.env.BLOCKCHAIN_SERVICE_AUTHORIZATION_USERNAME}:${process.env.BLOCKCHAIN_SERVICE_AUTHORIZATION_PASSWORD}`).toString('base64');
+
+const EVERY_SECOND = "*/1 * * * * *";
+
+// Set up default axios configuration
+const defaultOptions = {
+  baseURL: blockchainApiURL,
+  headers: {
+    'Authorization': `Basic ${authorizationToken}`,
+    'Content-Type': 'application/json'
+  }
+};
+
+// Create axios instance
+const axiosInstance = axios.create(defaultOptions);
+
+/**
+ * @summary Handles all blockchain service related functionality
+ * @see API Docs for further information
+ */
+class BlockchainServiceAPIClient {
+
+  /**
+   * @param {string} userAddress - wallet address
+   * @summary Authorizes user's wallet address to make calls 
```

Comment:
`Authorizes user's wallet address to make calls` -> `Authorizes user's wallet address to make blockchain transactions`